spec/features/contact_centre_agent_creates_appointments_spec.rb

```diff
@@ -2,6 +2,10 @@
 require 'rails_helper'
 
 RSpec.feature 'Agent creates appointments' do
+  let :day do
```

Comment:
We generally use parens around the arguments to `let`... Also don't be afraid of the single line and braces style if the thing is short enough. 😉